src/System.Private.CoreLib/src/System/Enum.CoreRT.cs

```diff
@@ -60,33 +60,42 @@ public int CompareTo(object target)
             ref byte pThisValue = ref this.GetRawData();
             ref byte pTargetValue = ref target.GetRawData();
 
+            // Compare the values. Note that we're required to return 0/1/-1 for backwards compat.
```

Comment:
FWIW, I do not think we have treated the exact values returned by CompareTo as something that needs to be maintained for compatibility (it is bug-for-bug compatibility, not compatibility). E.g. I believe we have differences in the globalization.

If this is important for something, it is fine to normalize here. Comparing enums is not performance critical.

Is there something real that would be broken by this, or is it just some tests failing somewhere?

Reply:
Failing test. ProjectN AppCompat infrastructure is on the floor right now, so I don't know if there's real world impact.